xcodeproj/internal/bazel_integration_files/bazel_build.sh

```diff
@@ -177,6 +186,7 @@ log=$(mktemp)
   build \
   --config=$config \
   --color=yes \
+  ${toolchain:+--define=SWIFT_CUSTOM_TOOLCHAIN="$toolchain"} \
```

Comment:
note that this doesn't really work in a lot of ways last I tried

Reply:
What breaks?

Reply:
https://github.com/bazelbuild/rules_swift#building-with-custom-toolchains is front and center, so if it doesn't work 😬

Reply:
linking will probably fail because it doesn't know about the toolchain path

Reply:
Worked in my, very minimal, testing: https://app.buildbuddy.io/invocation/ae071594-1485-41bb-9b97-675b3d88598d

Reply:
At the very least you need `--action_env=TOOLCHAINS=toolchain_id` to make sure that wrapped_clang invocations can use the same toolchain. In some situations you'll also need `--host_action_env` with the same